packages/story-editor/src/app/story/useStoryReducer/reducers/combineElements.js

```diff
@@ -102,12 +99,9 @@ function combineElements(
   // If the element we're dropping into is not background, maintain link, too.
   if (!secondElement.isBackground) {
     propsFromFirst.push('link');
-    // If relevant, maintain border, too.
-    if (canMaskHaveBorder(secondElement)) {
-      propsFromFirst.push('border');
-      if (canSupportMultiBorder(secondElement)) {
-        propsFromFirst.push('borderRadius');
-      }
+    propsFromFirst.push('border');
+    if (canSupportMultiBorder(secondElement)) {
+      propsFromFirst.push('borderRadius');
```

Comment:
All masks can have borders now (at least all the masks we are using), so this `if` did nothing.